### PR TITLE
Stats check

### PR DESCRIPTION
**Pull request by utkarsharma2**

### CODE REVIEW

File: tests/operators/test_agnostic_stats_check.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+import astro.sql as aql
+from astro.sql.table import Table
+
+log = logging.getLogger(__name__)
+DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+
+
+class TestStatsCheckOperator(unittest.TestCase):
+    """
+    Test Boolean Check Operator.
```
Comment (dimberman) [inline tests/operators/test_agnostic_stats_check.py:27]:
```suggestion
    Test Stats Check Operator.
```

File: integration-tests/test_load.py [hunk 2]
```diff
@@ -1,14 +0,0 @@
-import timeit
```
Comment (dimberman) [inline integration-tests/test_load.py:1]: Why are you deleting this?

File: src/astro/sql/operators/agnostic_stats_check.py [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+        accepted_std_div: int,
+        threshold: float,
+    ) -> None:
+        self.name = name
+        self.columns_map = columns_map
+        self.accepted_std_div = accepted_std_div
+        self.threshold = threshold
+
+
+class ChecksHandler:
```
Comment (dimberman) [inline src/astro/sql/operators/agnostic_stats_check.py:34]: Placing this into its own class is a really great call. Will make it easier if we decide to change how we handle checks in the future.

File: src/astro/sql/operators/agnostic_stats_check.py [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+        )
+
+    def prepare_main_stats_snowflake_sql(self, main_table: Table):
+        select_expressions = []
+        for check in self.checks:
+            for key in check.columns_map:
+                stats_query = "stddev({key}) as {check_name}_{key}_stddev, avg({key}) as {check_name}_{key}_avg"
+                replacements = {"{key}": key, "{check_name}": check.name}
+                for key, val in replacements.items():
+                    stats_query = stats_query.replace(key, val)
```
Comment (dimberman) [inline src/astro/sql/operators/agnostic_stats_check.py:67]: Do we need to worry about the column names being SQL escapes as well?

**Review by dimberman — CHANGES_REQUESTED** [review]

File: src/astro/sql/operators/agnostic_stats_check.py (hunk 4 above)
Comment (utkarsharma2) [reply src/astro/sql/operators/agnostic_stats_check.py:67]: I'm not sure entirely, but they are used with 'AS' keyword anything besides an identifier should raise an syntax error.

File: integration-tests/test_load.py (hunk 2 above)
Comment (utkarsharma2) [reply integration-tests/test_load.py]: Accidentally deleted it. Adding it back.

Comment (utkarsharma2) [reply integration-tests/test_load.py]: Also, In current state this fails on my local. Does this work for you while running UT?

Comment (dimberman) [reply integration-tests/test_load.py]: This doesn't need to run, it's just a tool I use for scale testing. I can fix it in a different PR

**Review by dimberman — APPROVED** [review]

**Pull request merged**